lib/models/repository-states/present.js

```diff
@@ -610,9 +610,6 @@ export default class Present extends State {
         const bundle = await this.git().getStatusBundle();
         const results = await this.formatChangedFiles(bundle);
         results.branch = bundle.branch;
-        if (!results.branch.aheadBehind) {
```

Comment:
this is unused, I take it?  good catch.

Reply:
Yup! `getStatusBundle` will always return an empty `aheadBehind` with our versions of git and `what-the-status`.